src/linagora.esn.calendar/app/event-message/event-message-edition/event-message-edition.controller.js

```diff
@@ -1,83 +1,81 @@
-(function(angular) {
-  'use strict';
+'use strict';
```

Comment:
You should squash this commit into the 1st one "remove IIFE"

Reply:
I have tried but it makes a conflict that hard to resolve ( Need to reset almost the commits) So I separated into a dedicated commit. And I think it is understandable because the commit message is clear enough.